Central and Eastern European Online Library Journals.js

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
Shouldn't our own translators bew prefixed with `ubtue_`?

Reply:
This is the official upstream version from zotero/translator

Reply:
Ok.